packages/plugins/graphql/server/services/builders/filters/content-type.js

```diff
@@ -51,6 +48,23 @@ module.exports = ({ strapi }) => {
           else if (isRelation(attribute)) {
             addRelationalAttribute(t, attributeName, attribute);
           }
+          // Handle components
+          else if (isComponent(attribute)) {
+            const componentModel = strapi.getModel(attribute.component);
```

Comment:
Could you put everything related to adding a component field in an `addComponentAttribute` method in order to keep consistency with the other types?